fuel/app/classes/controller/media.php

```diff
@@ -38,26 +43,49 @@ public function get_render($asset_id, $size='original')
 		return '';
 	}
 
+	/*
+		public function get_import()
+		{
+			// Validate Logged in
+			if (\Service_User::verify_session() !== true) throw new HttpNotFoundException;
+
+			Css::push_group(['core', 'media_import']);
+			Js::push_group(['angular', 'jquery', 'materia', 'author', 'dataTables']);
+
+			$this->inject_common_js_constants();
+
+			$theme = Theme::instance();
+			$theme->set_template('layouts/main');
+			$theme->get_template()
+				->set('title', 'Media Catalog')
+				->set('page_type', 'import');
+
+			$theme->set_partial('footer', 'partials/angular_alert');
+			$theme->set_partial('content', 'partials/catalog/media');
+
+			return Response::forge($theme->render());
+		}
+	*/
+
```

Comment:
Remove commented code if it isn't required anymore (I still do this too - no worries)